lib/display/display.ts

```diff
@@ -173,12 +173,15 @@ export function displayIssues(
 
   if (hasIssues) {
     const vulnerabilityIssues = issues.filter(
-      (issue) => issue.type === 'vulnerability',
+      (issue) => issuesData[issue.issueId].type !== 'license',
```

Comment:
So it might return vulns that is not states as "vulnerability" but it is?
I thought only "licenses" and "vulnerability" was the valid options. But it makes sense, if there is other possible values.

Reply:
what makes "type" from issuesData better than issue.type?

Reply:
what happens if issuesData does not contain the issueID as a key, then we try to fetch type from undefined. Maybe ?-operator could help?

Reply:
I created a new [commit](https://github.com/snyk/snyk-cpp-plugin/pull/103/commits/58be20bbbdb135cc883e81620b3891c5cefbec10) with adding the ?-operator and also renaming` vulnerabilityIssues` to `nonLicenseIssues`- as `issues` is already taken and in fact by the filtering it is everything that is not license- as the type field is optional so when it is not license it is undefined and then `issuesData[issue.issueId]?.type !== 'license'` returns true